packages/components/addon/components/hds/modal/index.hbs

```diff
@@ -9,9 +9,13 @@
   {{did-insert this.didInsert}}
   {{focus-trap isActive=this.isOpen focusTrapOptions=(hash onDeactivate=this.onDismiss clickOutsideDeactivates=true)}}
 >
-  {{yield (hash Header=(component "hds/modal/header" id=this.id onDismiss=this.onDismiss))}}
-  {{yield (hash Body=(component "hds/modal/body"))}}
-  {{yield (hash Footer=(component "hds/modal/footer" onDismiss=this.onDismiss))}}
+  {{yield
```

Comment:
This is pretty consistent with how we do other components, yes?

Reply:
No, that's the point in discussion. We do a single yield when the order doesn't matter, but in this case it matters. We would be introducing an "anti-pattern", making it difficult in future taking decisions when to use a single yield or multiple yields.